splash/splash.vcxproj

```diff
@@ -117,6 +117,12 @@
   <ItemGroup>
     <ResourceCompile Include="splash.rc" />
   </ItemGroup>
+  <ItemGroup>

```

Comment:
Is this needed?

Reply:
You are right, this does not appear to be required for it to build and run, I'm not sure why the solution/project updater would add it. I will remove it.